sdk/tables/azure-data-tables/src/table_clients.cpp

```diff
@@ -67,40 +67,6 @@ TableServiceClient::TableServiceClient(
       std::move(perOperationPolicies));
 }
 
-TableServiceClient::TableServiceClient(
-    const std::string& serviceUrl,
-    std::shared_ptr<Azure::Data::Tables::Credentials::NamedKeyCredential> credential,
-    const TableClientOptions& options)
-    : m_url(Azure::Core::Url(serviceUrl))
-{
-  m_namedKeyCredential = credential;
-  std::vector<std::unique_ptr<Azure::Core::Http::Policies::HttpPolicy>> perRetryPolicies;
-  std::vector<std::unique_ptr<Azure::Core::Http::Policies::HttpPolicy>> perOperationPolicies;
-  perRetryPolicies.emplace_back(std::make_unique<TimeoutPolicy>());
-  perOperationPolicies.emplace_back(std::make_unique<ServiceVersionPolicy>(options.ApiVersion));
-  m_pipeline = std::make_shared<Azure::Core::Http::_internal::HttpPipeline>(
-      options,
-      _detail::TablesServicePackageName,
-      PackageVersion::ToString(),
-      std::move(perRetryPolicies),
-      std::move(perOperationPolicies));
-
-  TableClientOptions newOptions = options;
-  newOptions.PerRetryPolicies.emplace_back(
-      std::make_unique<Azure::Data::Tables::_detail::Policies::SharedKeyLitePolicy>(credential));
```

Comment:
We no longer need `SharedKeyLitePolicy` and can remove it:
https://github.com/Azure/azure-sdk-for-cpp/blob/384552adff3d6cdea46cdf8a345120b183bdcc1e/sdk/tables/azure-data-tables/src/private/policies/shared_key_lite_policy.hpp#L13-L14

Reply:
Tracked in https://github.com/Azure/azure-sdk-for-cpp/issues/6267